$MirOS: ports/security/nss/patches/patch-security_nss_lib_ckfw_builtins_nssckbi_h,v 1.3 2010/12/12 16:38:46 tg Exp $

	Raise along with FULLPKGNAME-nssckbi in Makefile
	when ports/essentials/ca-certificates/certdata.c changes.

	Sync with: ports/www/firesomething/patches/patch-security_nss_lib_ckfw_builtins_nssckbi_h

--- security/nss/lib/ckfw/builtins/nssckbi.h.orig	Thu May 21 19:50:05 2009
+++ security/nss/lib/ckfw/builtins/nssckbi.h	Thu Jul 22 14:31:06 2010
@@ -76,7 +76,7 @@
  */
 #define NSS_BUILTINS_LIBRARY_VERSION_MAJOR 1
 #define NSS_BUILTINS_LIBRARY_VERSION_MINOR 75
-#define NSS_BUILTINS_LIBRARY_VERSION "1.75"
+#define NSS_BUILTINS_LIBRARY_VERSION "1.75.0.5"
 
 /* These version numbers detail the semantic changes to the ckfw engine. */
 #define NSS_BUILTINS_HARDWARE_VERSION_MAJOR 1
